SwiftCharts/Views/ChartAreasView.swift

```diff
@@ -51,6 +51,10 @@ open class ChartAreasView: UIView {
     }
     
     fileprivate func show(path: UIBezierPath) {
+        guard colors.count > 0 else {
```

Comment:
Now please squash the commits and I'll merge the pull request. Here's a nice explanation of how to do this, in case: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit thanks again!